File: CS/SecondaryAxesXSample/Model/NwindDbContext.cs
namespace SecondaryAxesXSample {
    using System;
    using System.Data.Entity;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Linq;

    public partial class NwindDbContext : DbContext {
        public NwindDbContext()
            : base("name=NwindDbContext") {
        }

        public virtual DbSet<SalesPerson> SalesPersons { get; set; }

        protected override void OnModelCreating(DbModelBuilder modelBuilder) {
            modelBuilder.Entity<SalesPerson>()
                .Property(e => e.UnitPrice)
                .HasPrecision(10, 4);

            modelBuilder.Entity<SalesPerson>()
                .Property(e => e.ExtendedPrice)
                .HasPrecision(19, 4);
        }
    }
}

File: VB/SecondaryAxesXSample/Model/NwindDbContext.vb
Imports System.Linq
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity
Imports System

Namespace SecondaryAxesXSample

    Partial Public Class NwindDbContext
        Inherits DbContext

        Public Sub New()
            MyBase.New("name=NwindDbContext")
        End Sub

        Public Overridable Property SalesPersons() As DbSet(Of SalesPerson)

        Protected Overrides Sub OnModelCreating(ByVal modelBuilder As DbModelBuilder)
            modelBuilder.Entity(Of SalesPerson)().Property(Function(e) e.UnitPrice).HasPrecision(10, 4)

            modelBuilder.Entity(Of SalesPerson)().Property(Function(e) e.ExtendedPrice).HasPrecision(19, 4)
        End Sub
    End Class
End Namespace
